sympy/functions/special/bessel.py

```diff
@@ -593,6 +593,14 @@ def fdiff(self, argindex=2):
             self * (self.order + 1)/self.argument
 
 
+def _jn_orthopoly(n, z):
```

Comment:
This name appears misleading. Though the coefficients ``fn`` for the spherical Bessel functions are located in ``orthopolys`` they are not actually polynomials. Neither are they orthogonal in any reasonable way. Perhaps just plain ``_jn`` could be sufficient?